client/blocks/inline-help/contextual-help.js

```diff
@@ -16,34 +17,40 @@ const fallbackLinks = [
 	{
 		link: 'https://en.support.wordpress.com/do-i-need-a-website-a-blog-or-a-website-with-a-blog/',
 		post_id: 143180,
-		title: 'Do I Need a Website, a Blog, or a Website with a Blog?',
-		description:
-			'If you’re building a brand new site, you might be wondering if you need a website, a blog, or a website with a blog. At WordPress.com, you can create all of these options easily, right in your dashboard.',
+		title: translate( 'Do I Need a Website, a Blog, or a Website with a Blog?' ),
+		description: translate(
+			'If you’re building a brand new site, you might be wondering if you need a website, a blog, or a website with a blog. At WordPress.com, you can create all of these options easily, right in your dashboard.'
+		),
 	},
 	{
 		link: 'https://en.support.wordpress.com/business-plan/',
 		post_id: 134940,
-		title: 'Uploading custom plugins and themes',
-		description: 'Learn more about installing a custom theme or plugin using the Business plan.',
+		title: translate( 'Uploading custom plugins and themes' ),
+		description: translate(
+			'Learn more about installing a custom theme or plugin using the Business plan.'
+		),
 	},
 	{
 		link: 'https://en.support.wordpress.com/all-about-domains/',
 		post_id: 41171,
-		title: 'All About Domains',
-		description: 'Set up your domain whether it’s registered with WordPress.com or elsewhere.',
+		title: translate( 'All About Domains' ),
+		description: translate(
+			'Set up your domain whether it’s registered with WordPress.com or elsewhere.'
+		),
 	},
 	{
 		link: 'https://en.support.wordpress.com/start/',
 		post_id: 81083,
-		title: 'Quick-Start Guide',
-		description:
-			"Use our five-step checklist to get set up and ready to publish, no matter what kind of site you're building.",
+		title: translate( 'Quick-Start Guide' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Quick Start Guide' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).